docs/community/governance.md

```diff
@@ -73,28 +73,82 @@ long experience with both the project and the larger ecosystem. When the core
-- [Nicholas Sofroniew](https://github.com/sofroniewn)
-
-The SC membership is revisited every January. SC members who do
-not actively engage with the SC duties are expected to resign. New members are
-added by nomination by a core developer. Nominees should have demonstrated
+The napari steering council (SC) will be no less than three members and no
+more than five members, with a strong preference for an odd number to ensure
+a simple majority vote outcome is always possible, and a preference for five
+members to ensure a diversity of voices. A majority of the SC will not be
+employed by the same entity. One seat on the steering council is reserved for
+a member elected by the [Institutional and Funding Partner Advisory Council](institutional-and-funding-partners),
+as detailed below. This member need not be an existing core developer.
+
+The SC membership, including the Institutional and Funding Partner seat, is revisited every January.
+SC members who do not actively engage with the SC duties are expected to resign. New members for 
+vacant spots are added by nomination by a core developer. Nominees should have demonstrated
 long-term, continued commitment to the project and its [mission and values](mission-and-values). A
 nomination will result in discussion that cannot take more than a month and
 then admission to the SC by consensus. During that time deadlocked votes of the SC will
-be postponed until the new member has joined and another vote can be held.
-
-The napari steering council may be contacted at `napari-steering-council@googlegroups.com`.
-Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team.
+be postponed until the new member has joined and another vote can be held. The Institutional and Funding Partner seat
+is elected by the Institutional and Funding Partner Advisory Council.
+
+The SC may be contacted at `napari-steering-council@googlegroups.com`.
+Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team. For a list of the current steering council see our [About Us](team) page.
+
+### Institutional and funding partners
+
+The SC is the primary leadership body for napari. No outside institution,
+individual or legal entity has the ability to own or control the project
+other than by participating in napari as contributors, core developers, and
+SC members. However, because institutions can be an important source of
+funding and contributions for the project, it is important to formally
+acknowledge institutional participation in the project. We call institutions
+recognized in this way Institutional and Funding Partners.
+
+Institutions become eligible to become an Institutional Partner by employing
+individuals who actively contribute to napari as part of their official
+duties, or by committing significant funding to napari, as determined by the
+SC. Once an institution becomes eligible to become an Institutional Partner,
```

Comment:
highlighting a potential side benefit of this: this provides a reason for employers of existing napari contributors/core-devs to officially recognise work on napari as part of someones 'duties', potentially legitimising work on OSS more generally as 'useful work' 🥳